tests/unit/test_service_steps.py

```diff
@@ -707,6 +727,13 @@ def test_databrew_start_job_run_step_creation():
     }
 
 
+@patch.object(boto3.session.Session, 'region_name', 'us-east-1')
+def test_databrew_start_job_run_step_creation_wait_for_task_token_raises_error():
+    with pytest.raises(ValueError):
+        GlueDataBrewStartJobRunStep('Start Glue DataBrew Job Run - WaitForTaskToken',
+                                    integration_pattern=IntegrationPattern.WaitForTaskToken)
```

Comment:
nit: can we assert the message as well? I think `pytest.raises` accepts a `match=<message>` - [documentation](https://docs.pytest.org/en/stable/reference.html?highlight=raises#pytest.raises)

Reply:
Yes - adding a validation on the error message will ensure no breaking changes are introduced

Added error message assert in the last commit